msteams-platform/tabs/quickstarts/create-channel-group-tab-asp-net-core.md

```diff
@@ -1,7 +1,259 @@
+
+- Open App studio and select the `Manifest editor` tab.
+
+- Select the *Import an existing app* tile in the Manifest editor to begin updating the app package for your tab. Recall that the source code comes with its own pre-made manifest and the `.csproj file` contains code to create an app package when the application is built. The name of your app package is *tab.zip*. You can search your local machine's file explorer or switch to Visual Studio `Folder View` to find your zip file's location. It should be found here:
+
+```bash
+ /bin/Debug/netcoreapp2.2/tab.zip
+```
+
+- Upload `tab.zip` to App Studio.
+
+### Update your app package with Manifest editor
+
+- Once you've uploaded your tab into Teams, you'll need to configure it to show content.
+
+- Select the tile for your newly imported tab in the right panel of the Manifest editor welcome page.
+
+There's a list of steps in the left-hand side of the Manifest editor, and on the right, a list of properties that need to have values for each of those steps. Much of the information has been provided by your `manifest.json` file but there are a few fields that you'll need to update:
+
+#### Details: App details
+
+- Under *Identification* select  <text style="color:#6364A5">***Generate***</text> to replace the placeholder id with a required Microsoft-generated GUID for your tab.
+
+- In the **Developer information** field update the `Website URL` with your `ngrok` HTTPS URL.
+
+- In the **App URLs** field update the `Privacy statement` and `Terms of use` URLs with your `ngrok` HTTPS URL. Remember to include the */privacy* and */tou* parameters at the end of the URLs.
+
+#### Capabilities
+
+- Select ***Tabs***.
+
+- To clear placeholder URLs from Manifest editor, scroll to the bottom of the Tabs section (*Select a tab from the following list to include any additional domains*) select the (•••) button and choose &#x1F5D1; Delete.
+
+##### Team Tab
+
+- Under `Team Tab` select the (**&#8226;&#8226;&#8226;**) button under the `Tab configuration url` field and choose &#128393; `Edit`.
+
+- Update the `Configuration URL` with your `ngrok` HTTPS URL. Remember to include the */channelgroup* parameter at the end of the URL.
+
+#### Finish
+
+##### Domains and permissions
+
+The `Domains from your tabs` should contain only your ngrok URL without the HTTPS prefix&mdash;`y8urGr7pChan3Ta2b.ngrok.io/`.
+
+##### Test and distribute
+
+- Select  <text style="color:#6364A5">***Install***</text>.
+
+>[!IMPORTANT]
+>In the `Description` field on the right you'll see the following warning:<br><br>
+&#9888; "**The 'validDomains' array cannot contain a tunneling site...**" <br><br>**This warning can be ignored while you're testing your tab.**<br><br>
+After your channel/group has been uploaded to Microsoft teams, via ngrok, and successfully saved, you can view it in tabs gallery, add it to the tabs bar, and interact with it until your tunnel session ends .<br><br>
```

Comment:
Remove everything after the validDomains warning. There's no guarantee they will ever submit to the app store